client/src/graphs/compositions/usePersistentGraph/index.ts

```diff
@@ -1,13 +1,8 @@
-import type { Ref } from 'vue'
-import type {
-  GNode,
-  GEdge,
-  GraphOptions
-} from '@graph/types'
-import { useUserEditableGraph } from '@graph/compositions/useUserEditableGraph'
-import type { GraphSettings } from '@graph/settings'
-import type { GraphTheme } from '@graph/themes'
-import type { GraphEvent } from '@graph/events'
+import type { Ref } from "vue";
+import type { GNode, GEdge, GraphOptions } from "@graph/types";
+import { useUserEditableGraph } from "@graph/compositions/useUserEditableGraph";
+import type { GraphTheme } from "@graph/themes";
+import type { GraphEvent } from "@graph/events";
 
 /**
  * extends the useGraph interface to include capabilities for storing and retrieving a graph from local storage.
```

Comment:
we should change the jsdoc to get rid of the limitation section now that we dont track settings or themes